english/great-white/great-white.md

```diff
@@ -0,0 +1,14 @@
+At this point, you are swimming calmly out in the ocean.
+All of a sudden, the water turns very still and you feel
+in the presence of some larger creature.
+
+You assume that maybe there's a big fish or shark swimming
+near you...
+
+Getting very nervous.  Starting to breath anxiously...
+
+Maybe you're overreacting, but you're plenty scared!
+
+Do you quickly swim back to shore, do you 
+[set your mind free](../yoga/mind/yoga_mind.md), or 
+do you [Learn to breathe](../yoga/breathe/breathe.md)?
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.